### ISSUE TITLE
Not compatible with authentication ?

### ISSUE THREAD

**Issue by tricosmo** [perdy/flama#36]
Hi there,
I just try to plugin authentication from Starlette, but looks like this is not possible.
From Starlette doc
```
from starlette.authentication import requires

@requires('authenticated')
async def foobar(request, schema: Schema):
    pass
```
This will raise error `AttributeError: 'OpenAPIConverter' object has no attribute 'field2parameter'`  from `flama/schemas.py`
Can't omit first parameter `request` cause that is required from the `requires` decorator for authentication.
Any work around?
Thanks

**Comment by perdy** [comment]
It should be fixed with version 0.16.0, that error was due to an incompatible version of apispec (4.0.0) with previous flama version (0.15.0).

**Comment by tricosmo** [comment]
Thank you very much.
